### PR TITLE
DEMO-94 Update user schema

### PR DESCRIPTION
**Pull request by sseerrnn**
- Add accountName, bankName for Artist
- Add creditCardName, creditCardNumber, creditCardExpireDate, creditCardCVV 
- Add role

### CODE REVIEW

**Comment by pol-alones** [comment]
Will you create role decorator? It can be used to be added in each endpoint `@Role()`

File: src/users/schema/users.schema.ts [hunk 1]
```diff
@@ -57,6 +58,62 @@ export class User {
...
+  @ApiProperty({ example: 'SCB' })
+  @Prop({})
+  bankName: string;
+
+  /**
+   * credit card name
+   * @example Pol Wongnai
+   */
+  @ApiProperty({ example: 'Pol Wongnai' })
+  @Prop({})
```
Comment (pol-alones) [inline src/users/schema/users.schema.ts]: Add default option?

File: src/users/schema/users.schema.ts [hunk 2]
```diff
@@ -57,6 +58,62 @@ export class User {
...
+   * @example 000
+   * */
+  @ApiProperty({ example: '000' })
+  @Prop({})
+  creditCardCVV: number;
+
+  /**
+   * role of the user
+   * @example user
+   * */
```
Comment (pol-alones) [inline src/users/schema/users.schema.ts]: The format is wrong?

File: src/users/schema/users.schema.ts [hunk 3]
```diff
@@ -57,6 +58,62 @@ export class User {
...
+   * @example 0000000000000000
+   */
+  @ApiProperty({ example: '0000000000000000' })
+  @Prop({})
+  creditCardNumber: number;
+
+  /**
+   * credit card expire date
+   * @example 01/2021
+   **/
```
Comment (pol-alones) [inline src/users/schema/users.schema.ts]: */

File: src/users/schema/users.schema.ts [hunk 4]
```diff
@@ -57,6 +58,62 @@ export class User {
...
+  @Prop({})
+  creditCardCVV: number;
+
+  /**
+   * role of the user
+   * @example user
+   * */
+  @ApiProperty({ example: 'user' })
+  @Prop({ default: 'user', enum: Role })
+  role: string;
```
Comment (pol-alones) [inline src/users/schema/users.schema.ts]: `roles` and it should be array of string.

**Review by pol-alones — CHANGES_REQUESTED** [review]

File: src/users/schema/users.schema.ts (hunk 4 above)
Comment (sseerrnn) [reply src/users/schema/users.schema.ts]: wanna get many role in one user?

**Comment by sseerrnn** [comment]
> Will you create role decorator? It can be used to be added in each endpoint `@Role()`

I will it later in the other branch

**Comment by sseerrnn** [comment]
user schema
{
    "_id": "5ff4c9d8e4b0f8b8b8b8b8b8",
    "username": "Saveng58",
    "email": "save-000@hotmail.com",
    "registrationDate": "2021-01-01T00:00:00.000Z",
    "accountNumber": "0000000000",
    "bankName": "SCB",
    "roles": [
      "user"
    ]
  }

**Review by pol-alones — APPROVED** [review]
GJ, btw u need 2 approver to merge a PR :skull:

**Review by Tantawit — APPROVED** [review]

**Pull request merged**